docs/topics/sessions.rst

```diff
@@ -32,7 +32,7 @@ in your ``routing.py``:
 
         "websocket": SessionMiddlewareStack(
             URLRouter([
-                url(r"^front(end)/$", consumers.AsyncChatConsumer),
+                url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
```

Comment:
While we're here. Url->re_path?

I'd be happy to take an action to conduct a general review for URL() if that'd help.

Reply:
Yes, these should all be `re_path`. (Not `path` if nested in a middleware...) Thanks!